src/components/BaseNavigation.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+	<nav class="navigation">
+		<ul class="list">
+			<li
+				v-for="(tab, index) in tabs"
+				:key="tab"
+				class="list-item"
+				:class="index < tabs.length - 1 ? 'border' : ''"
```

Comment:
could you move such logic to `computed` 
P.S `computed` can't get `arguments` but can return a `function`))